### PR TITLE
Fix IE syntax error

### PR DESCRIPTION
**Pull request by chugunomi**
Fixing issue #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,15 +12,15 @@ rangePlugin.install = function(Vue, options) {
         step = step || 1;
 
         if(typeof start !== 'number') {
-          console.warn(`Vue-for-range: provided argument for start - ${start} - is not a number.`)
+          console.warn('Vue-for-range: provided argument for start - ${start} - is not a number.')
```
Comment (krizzu) [inline index.js]: I've used [Temple literal syntax here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), so to make it work the same, please use string concatenation.

('Vue-for-range: provided argument for start -' + start + '- is not a number.')

File: index.js [hunk 2]
```diff
@@ -12,15 +12,15 @@ rangePlugin.install = function(Vue, options) {
...
           return [];
         }
         if(typeof end !== 'number') {
-          console.warn(`Vue-for-range: provided argument for end - ${end} - is not a number.`)
+          console.warn('Vue-for-range: provided argument for end - ${end} - is not a number.')
```
Comment (krizzu) [inline index.js]: Same as above

File: index.js [hunk 3]
```diff
@@ -12,15 +12,15 @@ rangePlugin.install = function(Vue, options) {
...
           return [];
         }
         if(typeof step !== 'number') {
-          console.warn(`Vue-for-range: provided argument for step - ${step} - is not a number.`)
+          console.warn('Vue-for-range: provided argument for step - ${step} - is not a number.')
```
Comment (krizzu) [inline index.js]: Same as above

**Review by krizzu — CHANGES_REQUESTED** [review]
Great, thanks for that! Just a small change is required and we're good to merge this :)

File: index.js (hunk 1 above)
Comment (chugunomi) [reply index.js]: Didn't noticed that, my bad 🙂

**Review by krizzu — APPROVED** [review]

**Pull request merged**